packages/manager/apps/web/client/app/hosting/cdn/shared/add-edit-cache-rule/hosting-shared-add-edit-cache-rule.routing.js

```diff
@@ -0,0 +1,52 @@
+import assign from 'lodash/assign';
+
+export default /* @ngInject */ ($stateProvider) => {
+  const resolve = {
+    goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+
+    rules: /* @ngInject */ ($transition$) => $transition$.params().rules,
+
+    priority: /* @ngInject */ ($transition$) => $transition$.params().priority,
+
+    callbacks: /* @ngInject */ ($transition$) =>
+      $transition$.params().callbacks,
+  };
+  const resolveEdit = {
+    rule: /* @ngInject */ ($transition$) => $transition$.params().rule,
+  };
+  const params = {
+    rules: null,
+    priority: null,
+    callbacks: null,
+  };
+  const paramsEdit = {
+    rule: null,
+  };
+
+  $stateProvider.state('app.hosting.cdn.shared.addCacheRule', {
+    url: '/add-cache-rule',
+    views: {
+      modal: {
+        component: 'hostingSharedSettingsAddCacheRule',
+      },
+    },
+    params,
+    layout: 'modal',
+    resolve,
+    atInternet: {
+      rename: 'web::hosting::cdn::configure::create-rule',
+    },
+  });
+
+  $stateProvider.state('app.hosting.cdn.shared.editCacheRule', {
+    url: '/edit-cache-rule',
+    views: {
+      modal: {
+        component: 'hostingSharedSettingsAddCacheRule',
+      },
+    },
+    params: assign({}, params, paramsEdit),
```

Comment:
lodash use doesn't seem necessary here 
```suggestion
    params: ({
    ... params,
    ...paramsEdit
    })
```
or 
```suggestion
    params: Object.assign({}, params, paramsEdit)
```